source/claim.ts

```diff
@@ -32,6 +38,14 @@ export interface ClaimSnakTimeValue {
 	};
 }
 
+export interface ClaimSnakQuantityValue {
+	readonly type: 'quantity';
+	readonly value: {
+		readonly amount: string;
+		readonly unit: number;
```

Comment:
Checking the data `unit` seems to be string and not a number?
https://www.wikidata.org/w/api.php?action=wbgetentities&ids=Q517354&format=json&languages=en&props=claims

```suggestion
		readonly unit: string;
```